packages/esbuild/src/stylable-esbuild-plugin.ts

```diff
@@ -532,7 +532,7 @@ function sortMarkersByDepth(
     const sorted = sortModulesByDepth(
         extracted,
         (m) => m.depth,
-        (m) => m.path,
+        (_) => '', // we don't want to sort by path this way we get similar results as the js injection mode
```

Comment:
sort or not sort, lets try and align CLI and other integrations, runtime/static to produce the same deterministic order